core/tests/webdriverio_utils/BlogDashboardPage.js

```diff
@@ -208,13 +192,17 @@ var BlogDashboardPage = function() {
 
   this.getMatTab = async function(tabName) {
     await waitFor.visibilityOf(
-      matTabLabels.first(), 'Mat Tab Toggle options take too long to appear.');
-    for (i = 0; i < await matTabLabels.count(); i++) {
-      var matTab = matTabLabels.get(i);
+      matTabLabel, 'Mat Tab Toggle options take too long to appear.');
+    var matTabLabels = await matTabLabelsSelector();
+    for (i = 0; i < matTabLabels.length; i++) {
+      var matTab = matTabLabels[i];
       var tabText = await action.getText(
         `Blog Dashboard tab ${i}`, matTab);
       if (tabText.startsWith(tabName)) {
         await action.click(`${tabName} tab`, matTab);
+        await waitFor.visibilityOf(matInkBar);
```

Comment:
No error message is given here. Please add an error message.

Reply:
Done.